tests/antares/services/local_services/test_study.py

```diff
@@ -2099,7 +2099,7 @@ def test_constraints_ini_have_correct_default_content(
         self, local_study_with_constraint, test_constraint, default_constraint_properties
     ):
         # Given
-        expected_ini_contents = """[0]
+        expected_ini_contents = """[test constraint]
```

Comment:
Actually we should have 0 here it's the weird convention of this ini file, the section is the number of the constraint

Same goes for the other tests you modified

Reply:
Yes didnt know about this specifity for bc